doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc

```diff
@@ -9,11 +9,27 @@ image::admin-and-config/task-inbox.png[Task inbox]
 
 Users with the `process-admin` role can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
```

Comment:
@kaldesai Users with the `process-admin` role  --> Process administrators
I try here to avoid talking about process-admin role and then, at the 'Note' part define how the process administrators can be specified, wdyt?

Reply:
Ack